runtime/Cpp/runtime/src/atn/ParserATNSimulator.cpp

```diff
@@ -451,7 +451,7 @@ std::unique_ptr<ATNConfigSet> ParserATNSimulator::computeReachSet(ATNConfigSet *
 
   // First figure out where we can reach on input t
   for (auto &c : closure_->configs) {
-    if (is<RuleStopState *>(c->state)) {
+    if (c->state->getStateType() == ATNState::RULE_STOP) {
```

Comment:
This is something I wanted to change for quite some time already! Now make it a `switch`...